code42.json

```diff
@@ -1925,6 +1925,505 @@
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "close case",
+            "identifier": "close_case",
+            "description": "Change the status of a Code42 case to 'CLOSED'",
+            "type": "investigate",
+            "read_only": false,
+            "parameters": {
+                "case_number": {
+                    "description": "Case number",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "case number"
+                    ],
+                    "order": 0
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.data.*.number",
+                    "data_type": "string",
+                    "contains": [
+                        "case number"
+                    ],
+                    "column_name": "Case number",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.data.*.name",
+                    "data_type": "string",
+                    "column_name": "Case name",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.data.*.description",
+                    "data_type": "string",
+                    "column_name": "Description",
+                    "column_order": 2
+                },
+                {
+                    "data_path": "action_result.data.*.subjectUsername",
+                    "data_type": "string",
+                    "contains": [
+                        "email",
+                        "user name"
+                    ],
+                    "column_name": "Subject",
+                    "column_order": 3
+                },
+                {
+                    "data_path": "action_result.data.*.assigneeUsername",
+                    "data_type": "string",
+                    "contains": [
+                        "email",
+                        "user name"
+                    ],
+                    "column_name": "Assignee",
+                    "column_order": 4
+                },
+                {
+                    "data_path": "action_result.data.*.findings",
+                    "data_type": "string",
+                    "column_name": "Findings",
+                    "column_order": 5
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
```

Comment:
I think should be the case number now

```
"data_path": "action_result.summary.case_number",
"data_type": "numeric"
```

Reply:
I don't see any functional difference in the phantom UI when I make this change, or even if I delete this object from the output altogether... Where do these outputs that aren't columns actually show up?

Reply:
@unparalleled-js may know more but I believe that the purpose is for them to be consumed in a playbook context.
https://docs.splunk.com/Documentation/Phantom/4.10.4/PlaybookAPI/Datapaths